translation/source/coordinates.xml

```diff
@@ -13,4 +13,5 @@
   <string name="startTraining">Start training</string>
   <string name="findSquare">Find square</string>
   <string name="nameSquare">Name square</string>
+  <string name="disableTime">Disable</string>
```

Comment:
On second thought, we can avoid the need for a translation altogether by just using the infinity symbol, ∞. Might be the cleanest way.

Reply:
OK, I'm updating.